pkg/minikube/drivers/hyperkit/disk.go

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package hyperkit
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"syscall"
+
+	"github.com/cloudflare/cfssl/log"
+	"github.com/docker/machine/libmachine/mcnutils"
+)
+
+func createDiskImage(sshKeyPath, diskPath string, diskSize int) error {
+	tarBuf, err := mcnutils.MakeDiskImage(sshKeyPath)
+	if err != nil {
+		return err
+	}
+
+	file, err := os.OpenFile(diskPath, os.O_CREATE|os.O_EXCL|os.O_WRONLY, 0644)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+	file.Seek(0, os.SEEK_SET)
+
+	if _, err := file.Write(tarBuf.Bytes()); err != nil {
+		return err
+	}
+	file.Close()
+
+	if err := os.Truncate(diskPath, int64(diskSize*1048576)); err != nil {
```

Comment:
diskSize * 10^6 instead of diskSize * 2^20.  This code will give you diskSize * MiB instead of disksize * MB

I tried this out with 

```golang
package main

import "os"

func main() {
	os.Create("./test")
	os.Truncate("./test", 1*1048576)
}
```

```
$ ls -l --block-size=KB
total 5kB
-rw-r----- 1 mrick eng    1kB Aug 22 13:16 main.go
-rw-r----- 1 mrick eng 1049kB Aug 22 13:16 test
```